docker/falco/Dockerfile

```diff
@@ -19,17 +19,26 @@ RUN cp /etc/skel/.bashrc /root && cp /etc/skel/.profile /root
 
 RUN apt-get update \
 	&& apt-get install -y --no-install-recommends \
+	bc \
+	bison \
 	ca-certificates \
 	clang \
 	curl \
 	dkms \
+	dwarves \
+	flex \
 	gcc \
 	gcc-11 \
 	gnupg2 \
 	jq \
-	libelf1 \
+	libc6-dev \
+	libelf-dev \
```

Comment:
I know `libelf-dev` is useful for OpenShift compatibility but can we understand why the others? This is because we may be tempted to remove them in the future and it'd be good to have any kind of "trail" we can follow to understand why something was there in the first place.

Reply:
These were used in the `-legacy` image. 
My opinion is: using the ones used by `-legacy` image we are sure that we are not breaking an existing behavior; perhaps we miss some more, but at least the 2 images are equal now.

Reply:
Let me take a bit of a closer look. I don't think we want a bloated image but at the same time we want to increase compatibility.